### PR TITLE
Remove qs support and hard deprecate `qs_from_url`

### LINKED ISSUE #298: Drop qs dependency
Because maintainers are reckless individuals who cannot be relied upon. 

qs has been "deprecated" without warning and it has been removed from CRAN which broke the entire nflverse automation structure. 

https://github.com/qsbase/qs/issues/103

### PR DESCRIPTION
**Pull request by mrcaseb**
closes #298 

- hard deprecated `qs_from_url`, i.e. it errors when called (we'll remove it in a future release)
- no more qs import or remote
- removed qs from all load_functions argument checks
- qs option raises packageStartupMessage
- adjusted tests to either expect an error or just drop qs usage
- removed qs from readme
- kept `qs_from_url` in `clear_cache` (for now) to allow users to get rid of cached qs files. (I guess this is an edge case where somebody used the filesystem cache with a long reset time).

### CODE REVIEW

File: R/from_url.R [hunk 1]
```diff
@@ -200,32 +206,10 @@ parquet_from_url <- function(url) {
...
-    )
-
-    return(data.table::data.table())
-  }
-
-  data.table::setDT(content)
-  return(content)
+  lifecycle::deprecate_stop(
+    "1.6.0", "qs_from_url()",
+    details = "Please use on of the other `*_from_url()` functions."
```
Comment (tanho63) [inline R/from_url.R]:
```suggestion
    details = "Please use one of the other `*_from_url()` functions."
```

**Review by tanho63 — DISMISSED** [review]

**Review by tanho63 — APPROVED** [review]
